pkg/services/ngalert/api/tooling/Makefile

```diff
@@ -8,6 +6,9 @@ GO_PKG_FILES = $(shell find $(API_DIR) -name *.go -print)
 PATH_DOWN = pkg/services/ngalert/api/tooling
 PATH_UP = ../../../../..
 
+GO ?= $(shell which go)
+SWAGGER = GODEBUG=gotypesalias=0 $(GO) tool swagger
```

Comment:
The reason for `GODEBUG=gotypesalias=0` is described in https://github.com/grafana/grafana/pull/102434